src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -440,6 +442,9 @@ void CommissionerDiscoveryController::InternalHandleContentAppPasscodeResponse()
             }
             CharSpan rotatingIdSpan(rotatingIdBuffer, 2 * rotatingIdLength);
 
+            // Store rotating ID as tempAccountIdentifier to use in AccountLogin::Login payload later.
+            mTempAccountIdentifier = rotatingIdSpan;
```

Comment:
I suggest grabbing this from the client at the time when you need it rather than here since this only gets called when we get the passcode back from the content app. 

The platform should invoke the Login command after commissioning, regardless of whether the content app provided the passcode or not.

Reply:
Are you saying we want to send the Login command on successfull commissioning in all cases - regardless if the setup PIN prompt was shown or not?

Reply:
Please see updated question regarding this [here](https://github.com/project-chip/connectedhomeip/pull/34162/files#diff-0619e2ba03ec92e4032b69b6a7c5704aee9b14df0da72d9664d8e3b8879ffb13R766)